pyseed/seedclient.py

```diff
@@ -253,7 +246,7 @@ def _raise_error(self, response, error_msg, stack_pos=0, *args, **kwargs):
 
     def _set_params(self, params):
         """Add org_id"""
-        params['org_id'] = self.org_id
+        params['organization_id'] = self.org_id
```

Comment:
Is the SEED v2 API being done away with entirely?
If not, what happens if you send 'organization_id', instead of 'org_id', to a v2 endpoint?

Reply:
This was actually the main issue - the current release of py-seed is broken because passing `org_id` fails with the v2 endpoints.  We'll keep the v2 API for the foreseeable future, but I went ahead and updated the endpoints to v3 because they have much more logical/consistent patterns (that was optional).

Reply:
That's odd; I have been using this client with SEED v2.6.1 with v2 endpoints without issue for some time.

Even though the init accepts and urls_map that means changing the URLs to v3 doesn't technically break the ability to use it with v2 URLs, I would also like to propose an alternate to just changing the URLs that should make it more robust and easier on current v2 users (like me).
I'm going to submit my suggested change as a pull request against this branch.

Reply:
Sounds good - I only tested one endpoint with py-seed against 2.6.1 and it was failing:
```python
seed_client.list(data_name='results', endpoint='properties', **{'per_page': 5})
```

Returns: `pyseed.exceptions.SEEDError: SEEDError: Need to pass organization_id as query parameter`

Reply:
Ahh.  I think that explains it.  When Paul Munday and I did the work in SEED way back we set up an OrgMix and created ViewSet classes based off of it that use:
```
def get_org_or_id(dictlike):
    """Get value of organization or organization_id"""
    # while documentation should encourage the use of one consistent key choice
    # for supplying an organization to query_params, we check all reasonable
    # permutations of organization id.
    org_query_strings = ['organization', 'organization_id', 'org_id', 'org']
    org_id = None
    for org_str in org_query_strings:
        org_id = dictlike.get(org_str)
        if org_id:
            # Type case the organization_id as a integer
            if '_id' in org_str:
                org_id = int(org_id)
            break
    return org_id
```
Which you can see is more forgiving of the organization query params.  But that particular PropertyViewSet doesn't utilize those classes.  
On the bright side, having dug through to find that, it would seem that all the v2 endpoints that allowed 'org_id' will still be good with 'organization_id'